xml/System.Net.Http/HttpClientHandler.xml

```diff
@@ -515,8 +515,8 @@ After NuGet package v4.3.2, the default value of <xref:System.Net.DecompressionM
         <ReturnType>System.Collections.Generic.IDictionary&lt;System.String,System.Object&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets a set of properties for the all HttpClient requests.</summary>
```

Comment:
grammar: "for the all" -> "for the"

While this property is only a getter and not a setter, it gets a dictionary object.  The dictionary object can then be used to set and get entries based on string indexes.  

By default the property "bag" is empty.  But developers can use this property bag, set entries into it and then use this dictionary in custom handlers for to do special processing.